website/docs/CBD.md

```diff
@@ -0,0 +1,112 @@
+---
+slug: cbd
+---
+
+# Condition Based Decryption
+
+Condition Based Decryption (CBD) is a programmable access control service, in which decryption rights are based on the verified fulfillment of predefined conditions.
+Access conditions can be EVM-based (e.g. does the requester own this NFT?), RPC-driven (e.g. has the requester sent more than N transactions?) or time-based (e.g. has a preset period elapsed, after which the recipient's requests will be ignored).
+These conditions are also composable and can be combined in any logical sequence or decision tree.  
+
+CBD involves splitting a joint secret (a decryption key) into multiples _shares_ and distributing those among authorized and collateralized node operators ([Stakers](https://threshold.network/earn/staker) in the Threshold network).
+A minimum number – a _threshold_ – of those operators holding the key shares must be online and actively participate in partial decryptions that can subsequently be combined by the requester to reconstruct the original plaintext data.
+
+## Build a Cohort
+A [Cohort](./cohort) defines the collection of nodes that will provide CBD services.
+_Threshold_ and _Shares_ are two parameters used to construct a Cohort.
+For example, a `3-of-5` Cohort requires a threshold of 3 out of 5 shares to reconstruct the original plaintext data.
+
+We can define a cohort by:
+
+```js
+import { Cohort } from '@nucypher/nucypher-ts'
+
+const config = {
+    threshold: 3,
+    shares: 5,
+    porterUri: 'https://porter-ibex.nucypher.community',
+};
+const newCohort = await Cohort.create(config);
+```
+
+We also provide `porterUri`.
+[Porter](./Glossary.md#porter) is a service that helps us interact with nodes.
+In this case, we are using an `ibex` porter which means we're on a testnet.
+
+## Create a Condition
+
+Condition Based Decryption obviously also needs some [Conditions](./conditions).
+In this tutorial, we will check that the user owns a specific ERC721 NFT.
+
+```js
+import { Conditions } from '@nucypher/nucypher-ts'
+
+const NFTOwnership = new Conditions.ERC721Balance({
```

Comment:
Is this supposed to be `Conditions.ERC721Ownership`, and not `ERC721Balance`? `ERC721.ownerOf` is specific NFT ownership, and `ERC721.balanceOf` is any NFT from the contract.

Additionally `ownerOf` takes the token id ("5954") whereas `balanceOf` takes an address - https://docs.openzeppelin.com/contracts/4.x/api/token/erc721#IERC721.